src/components/Square.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+const Square = ({
+  i,
+  squares,
+  setSquares,
+  remainingAlphabet,
+  setRemainingAlphabet,
+  errorMessage,
+  setErrorMessage,
+  numberSelected,
+}) => {
+  const editInput = e => {
+    const newSquares = JSON.parse(JSON.stringify(squares));
+    const workRemainingAlphabet = JSON.parse(JSON.stringify(remainingAlphabet));
+    let newLetter = e.target.value.replace(/[^a-z]/gi, '').toUpperCase();
+    // Check to see if you have reached the extent of your letter useage
+    let workErrorMessage = '';
+    if (
+      numberSelected < 27 - remainingAlphabet.length + 1 &&
```

Comment:
I'd try to avoid magic numbers like this. Like what makes 27 so special. Make it a constant with a name somewhere its purpose is documented

Reply:
Ok...... 27 has been defined as to what it represents.